tests/tests/test-fees/test-fee-multiplier.ts

```diff
@@ -107,6 +109,50 @@ describeDevMoonbeam("Max Fee Multiplier", (context) => {
     let amount = (withdrawEvent.event.data as any).amount.toBigInt();
     expect(amount).to.equal(1_500_000_012_598_000_941_192n);
   });
+
+  // similar to tests in test-contract-fibonacci.ts, which implements an Ethereum txn which uses
+  // most of the block gas limit. This is done with the fee at its max, however.
+  it("fibonacci[370] should be spendable", async function () {
+    let blockNumber = (await context.polkadotApi.rpc.chain.getHeader()).number.toBn();
+    let baseFeePerGas = BigInt( (await context.web3.eth.getBlock(blockNumber)).baseFeePerGas);
+    expect(baseFeePerGas).to.equal(125_000_000_000_000n);
+
+    const { contract, rawTx } = await createContract(context, "Fibonacci", { gasPrice: "0x"+baseFeePerGas.toString(16), });
+    const {
+      result: { hash: createTxHash },
+    } = await context.createBlock(rawTx);
+
+    let receipt = await context.web3.eth.getTransactionReceipt(createTxHash);
+    expect(receipt.status).to.be.true;
+
+    // the multiplier (and thereby base_fee) will have decreased very slightly...
+    blockNumber = (await context.polkadotApi.rpc.chain.getHeader()).number.toBn();
+    baseFeePerGas = BigInt( (await context.web3.eth.getBlock(blockNumber)).baseFeePerGas);
+    expect(baseFeePerGas).to.equal(124_880_845_878_351n);
+
+    const tx = await createContractExecution(
+      context,
+      {
+        contract,
+        contractCall: contract.methods.fib2(370),
+      },
+      { gasPrice: "0x"+baseFeePerGas.toString(16), },
```

Comment:
PSA: this value can be greater than Javascript's "max safe integer," which can lead to subtle failures...